mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/task/MailboxMergingTask.java

```diff
@@ -98,6 +104,70 @@ public void incrementFailedCount() {
         }
     }
 
+    private static class MailboxMergingTaskDTO implements TaskDTO {
+        private static final CassandraId.Factory CASSANDRA_ID_FACTORY = new CassandraId.Factory();
+
+        private final String type;
+
+        private final long totalMessageCount;
+        private final String oldMailboxId;
+        private final String newMailboxId;
+
+        public MailboxMergingTaskDTO(@JsonProperty("type") String type,
+                                     @JsonProperty("totalMessageCount") long totalMessageCount,
+                                     @JsonProperty("oldMailboxId") String oldMailboxId,
+                                     @JsonProperty("newMailboxId") String newMailboxId) {
+            this.type = type;
+            this.totalMessageCount = totalMessageCount;
+            this.oldMailboxId = oldMailboxId;
+            this.newMailboxId = newMailboxId;
+        }
+
+        public static MailboxMergingTaskDTO fromDTO(MailboxMergingTask domainObject, String typeName) {
```

Comment:
does it pass checkstyle? I thought it enforce static before non-static

Reply:
it does